Formula/fastfetch.rb

```diff
@@ -17,10 +17,14 @@ class Fastfetch < Formula
     sha256 x86_64_linux:   "8757d24b5daa28af495feec81f4be8fa5952c23df8f24185d8a0c69b5dc9269e"
   end
 
+  depends_on "chafa" => :build
   depends_on "cmake" => :build
+  depends_on "imagemagick" => :build
```

Comment:
Are you sure this is never used at runtime? That's very unusual.

Reply:
[All runtime dependencies in fastfetch are dynamically loaded with `dlopen`](https://github.com/LinusDierheimer/fastfetch#dependencies).  I didn't make it a hard dependency because they are rarely used. People can install the dependencies if they need them.

Reply:
So it's a runtime dependency then.

Reply:
If a dependency cannot be found at runtime, the feature that uses the dependency is disabled, which doesn't make the whole program unusable.